src/utils/stripe/getStripe.tsx

```diff
@@ -3,9 +3,10 @@ import { Stripe, loadStripe } from '@stripe/stripe-js';
 let stripePromise: Promise<Stripe | null>;
 
 const getStripe = (paymentSetup:any) => {
+  let lang = localStorage.getItem('language') || 'en';
   const key = paymentSetup?.gateways?.stripe?.authorization.stripePublishableKey ? paymentSetup?.gateways?.stripe?.authorization.stripePublishableKey : paymentSetup?.gateways?.stripe?.stripePublishableKey;
   const account = paymentSetup?.gateways?.stripe?.authorization.accountId;
-  stripePromise = loadStripe(key,{stripeAccount:account});
+  stripePromise = loadStripe(key,{stripeAccount:account,locale:lang});
```

Comment:
I tested this before and I saw no difference from not doing it. Stripe is already loading some translation resources based by the browser language and some translations (which maybe are only for development purpose) do not seem to get translated:

<img width="413" alt="Bildschirmfoto 2021-03-03 um 10 51 53" src="https://user-images.githubusercontent.com/1532418/109787883-fbc04400-7c0e-11eb-8680-28323314ea9d.png"><img width="407" alt="Bildschirmfoto 2021-03-03 um 10 40 59" src="https://user-images.githubusercontent.com/1532418/109787887-fcf17100-7c0e-11eb-9d5a-d777e414644c.png">

But for the case that someone wants to use our website in another language as the default language of his browser, we should keep this patch anyway.